x-pack/plugins/apm/public/components/app/service_overview/index.tsx

```diff
@@ -54,6 +61,10 @@ export function ServiceOverview() {
     isSingleColumn,
   };
 
+  const isLoading =
+    serviceAgentStatus === FETCH_STATUS.LOADING ||
+    serviceAgentStatus === FETCH_STATUS.NOT_INITIATED;
```

Comment:
Everytime I see this line (we have it in many places) I think about adding a helper. 

```suggestion
  const isLoading = isLoading(serviceAgentStatus)
```

We can add the helper to `use_fetcher`:
```
function isLoading(fetchStatus: FETCH_STATUS) {
  return fetchStatus === FETCH_STATUS.LOADING || fetchStatus === FETCH_STATUS.NOT_INITIATED;
}
```

Reply:
I updated all places where we use both status to indicate that it is loading.